.github/workflows/check-version.yml

```diff
@@ -0,0 +1,87 @@
+name: Check version
+
+on:
+  push:
+    branches:
+      - develop
+      - release/*
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  VERSION_FILE: ./package.json
+
+jobs:
+  check-version:
+    name: Check version
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Get current version
+        id: current_version
+        run: |
+          currentVersion=$(node -p -e "require('${{ env.VERSION_FILE }}').version")
+
+          echo "version=$currentVersion" >> $GITHUB_OUTPUT
+
+      - name: Get latest version
+        id: latest_version
+        uses: pozetroninc/github-action-get-latest-release@v0.7.0
+        with:
+          repository: nimblehq/infrastructure-templates
```

Comment:
We should better run this workflow on the `main` branch, get and bump the new version base on the current version from the main 💡 
Running on the `main` can help us ensure we can fetch the current version even when we have no deployment. Also, getting a version from Firebase could be hard for Android or iOS to reuse the workflow.

Reply:
This is a good idea, I changed to using it in https://github.com/nimblehq/infrastructure-templates/pull/237/commits/3b0ed365bfb509d6ebd0f30f4d99a58704f62951